### PR TITLE
[AIRFLOW-173] - Initial implementation of FileSensor

### PR DESCRIPTION
**Pull request by trixpan**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-173

Implement a simple FileSensor to detect the creation of files and folder
in a given filesystem (a la HDFSSensor)

### CODE REVIEW

**Comment by trixpan** [comment]
@bolkedebruin any chance of reviewing this PR?

Cheers

File: airflow/hooks/fs_hook.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (bolkedebruin) [inline airflow/hooks/fs_hook.py]: This file needs to go to contrib. As does its operator

**Comment by bolkedebruin** [comment]
Good point. Please add a couple of tests and move the files to contrib. Then LGTM

**Comment by bolkedebruin** [comment]
@trixpan any update?

**Comment by trixpan** [comment]
@bolkedebruin apologies for the delay, long weekend. :-)

Moved to contrib.

**Comment by trixpan** [comment]
@bolkedebruin PR passes travis and should be ready for review.

**Comment by bolkedebruin** [comment]
+1

**Review by Anmolk7 — COMMENTED** [review]
the FileSensor doesnt sense file. The task terminates as success even though there is no filepath given or a non existing filepath is given to the extra field in the connections

**Pull request closed without merging**